ister_test.py

```diff
@@ -4226,6 +4225,175 @@ def mock_sleep(sec):
     commands_compare_helper(commands)
 
 
+def gui_find_current_disk_success():
+    """
+    Finds the disk where root is currently mounted
+    """
+    import subprocess
+
+    global COMMAND_RESULTS
+    COMMAND_RESULTS = []
+
+    def mock_check_output(cmd):
+        COMMAND_RESULTS.extend(cmd)
+        return u"NAME MOUNTPOINT\n" \
+               "sda\n"             \
+               "sda1\n"            \
+               "sda2 [SWAP]\n"     \
+               "sda3 /\n".encode('utf-8')
+
+    commands = ["lsblk", "-l", "-o", "NAME,MOUNTPOINT"]
+    check_output_backup = subprocess.check_output
+    subprocess.check_output = mock_check_output
+    chooseact = ister_gui.ChooseAction()
+    result = chooseact._find_current_disk()
+    subprocess.check_output = check_output_backup
+    if not result:
+        raise Exception("Unable to find root disk")
+
+    commands_compare_helper(commands)
+
+
+def gui_find_current_disk_failure():
+    """
+    Returns an empty string when root disk cannot be found
+    """
+    import subprocess
+
+    global COMMAND_RESULTS
+    COMMAND_RESULTS = []
+
+    def mock_check_output(cmd):
+        COMMAND_RESULTS.extend(cmd)
+        return "NAME MOUNTPOINT\n" \
+               "sda\n"             \
+               "sda1\n"            \
+               "sda2 [SWAP]\n"     \
+               "sda3\n".encode('utf-8')
+
+    commands = ["lsblk", "-l", "-o", "NAME,MOUNTPOINT"]
+    check_output_backup = subprocess.check_output
+    subprocess.check_output = mock_check_output
+    chooseact = ister_gui.ChooseAction()
+    result = chooseact._find_current_disk()
+    subprocess.check_output = check_output_backup
+    if result:
+        raise Exception("Found root disk when there was none to be found")
+
+    commands_compare_helper(commands)
+
+
+def gui_mount_host_disk():
```

Comment:
This is a good test but misses bits that may be needs another test as if you didn't have a EFI partition on sdc it would have used sda instead.